ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json

```diff
@@ -119,6 +150,622 @@
+      "post": {
+        "sources": [
+          {"name": "OUTPUT", "type": "table", "schema": "ID INT KEY, RP INT, RO BIGINT"}
+        ]
+      }
+    },
+    {
+      "name": "Select ROWPARTITION/ROWOFFSET from CTAS",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": true
+      },
+      "statements": [
+        "CREATE TABLE input (id INT PRIMARY KEY, foo STRING) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE TABLE queryable AS SELECT id, foo FROM input;",
+        "CREATE TABLE output AS SELECT id, ROWPARTITION AS rp, ROWOFFSET AS ro FROM queryable;"
+      ],
+      "inputs": [
+        {"topic":  "test_topic", "key": 1, "value": {"foo": "a"}, "timestamp": 5},
+        {"topic":  "test_topic", "key": 2, "value": {"foo": "b"}, "timestamp": 6},
+        {"topic":  "test_topic", "key": 3, "value": {"foo": "c"}, "timestamp":  7}
+      ],
+      "outputs": [
+        {"topic":  "OUTPUT", "key": 1, "value": {"RP": 0, "RO": 0}, "timestamp": 5},
+        {"topic":  "OUTPUT", "key": 2, "value": {"RP": 0, "RO": 1}, "timestamp": 6},
+        {"topic":  "OUTPUT", "key": 3, "value": {"RP": 0, "RO": 2}, "timestamp": 7}
+      ],
+      "post": {
+        "sources": [
+          {"name": "OUTPUT", "type": "table", "schema": "ID INT KEY, RP INT, RO BIGINT"}
+        ]
+      }
+    },
+    {
+      "name": "join on ROWPARTITION - INCOMPLETE",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": true
+      },
+      "statements": [
+        "CREATE STREAM a (id INT KEY, foo STRING) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM b (id INT KEY, foo STRING) WITH (kafka_topic='test_topic_2', value_format='JSON');",
+        "CREATE STREAM output AS SELECT a.ROWPARTITION AS rpa, a.ROWOFFSET AS roa, b.ROWPARTITION as rpb, b.ROWOFFSET as rob FROM a JOIN b ON a.ROWPARTITION = b.ROWPARTITION;"
+      ],
+      "inputs": [
+        {"topic":  "test_topic", "key": 1, "value": {"foo": "a"}, "timestamp": 5},
+        {"topic":  "test_topic", "key": 2, "value": {"foo": "b"}, "timestamp": 6},
+        {"topic":  "test_topic", "key": 3, "value": {"foo": "c"}, "timestamp":  7},
+        {"topic":  "test_topic_2", "key": 4, "value": {"foo": "d"}, "timestamp":  8},
+        {"topic":  "test_topic_2", "key": 5, "value": {"foo": "e"}, "timestamp":  9},
+        {"topic":  "test_topic_2", "key": 6, "value": {"foo": "f"}, "timestamp":  10}
+      ],
+      "outputs": [
+        {"topic":  "OUTPUT", "key": 1, "value": {"RP": 0, "RO": 0, "RT": 5}, "timestamp": 5},
+        {"topic":  "OUTPUT", "key": 2, "value": {"RP": 0, "RO": 1, "RT": 6}, "timestamp": 6},
+        {"topic":  "OUTPUT", "key": 3, "value": {"RP": 0, "RO": 2, "RT": 7}, "timestamp": 7}
+      ],
+      "post": {
+        "sources": [
+          {"name": "OUTPUT", "type": "stream", "schema": "ID INT KEY, RP INT, RO BIGINT, RT BIGINT"}
+        ]
+      }
+    },
+    {
+      "name": "join on ROWOFFSET - INCOMPLETE",
```

Comment:
What does "INCOMPLETE" mean in the name here?

Reply:
They weren't working and hadn't finished them- this could have been better communicated via a GitHub comment, but I thought I relayed to you I had some nonfunctioning tests that I pushed? Sorry if I didn't do that or didn't make it clear enough that these were the queries in question, I'll comment the titles in this situation next time instead of writing it into the code.